### DISCUSSION TITLE
Revoked ssl not raising exception with verify=True

### DISCUSSION THREAD

**Discussion by mmandric** [encode/httpx#1729] [Potential Issue]
````
    import httpx
    r = httpx.get('https://revoked.badssl.com/', verify=True)
    print(r.status_code)
    r = httpx.get('https://untrusted-root.badssl.com/', verify=True)
    print(r.status_code)
````
Running this code print 200 and raises an exception for the second request, but why is the first request returning 200 and not raising an exception?

**Comment by j178** [comment]
Related: https://github.com/psf/requests/issues/3770

**Reply by mmandric** [reply]
Thanks for that, I wasn't aware there was an issue about this, seems like it's going to remain this way.
